### ISSUE TITLE
Restore copies whole folder instead of file inside - Pokemon Let's Go Eevee

### ISSUE THREAD

**Issue by BadlyDrawnRhino** [J-D-K/JKSV#125]
I was trying to restore an edited Pokemon Let's Go Eevee save just now, and was puzzled as to why the game was loading the old version of the save prior to editing. I went into the file view mode for the game and noticed there was an extra file in there alongside the savedata.bin file, which happened to share the name of the backup I was trying to restore. So I moved my edited .bin file up one level out of the backup folder and into the main JKSV folder for the game and restored that directly and it worked. Hope that makes sense, I can so some screenshots if there's confusion.

**Comment by J-D-K** [comment]
I think I get what you're saying. It sounds like it's treating the folder as a file. JKSV is programmed to just copy files without the zip extension to the save of they're in the game directory as a precaution. It also explains why it didn't throw the empty save error. I'll get back to ya asap. I've been busy lately and today I'm pretty beat.

**Comment by BadlyDrawnRhino** [comment]
Hey, sorry I'm only just replying. No hurry, just wanted to make you aware of the issue. Now that I know the workaround, it'll be pretty simple to avoid this issue until you do have time to fix.

**Comment by J-D-K** [comment]
Don't worry about it. I'm still going to need ya soon to nail down exactly what's happening.

**Comment by fennectech** [comment]
A copy of their JKSV folder might help.  Even if it points out a glitch with folder creation

**Comment by J-D-K** [comment]
It might also be a path bug. I'm not really sure at the moment. If it were a bug with it creating a funky folder, it probably wouldn't be able to write his save inside of it, which it did.

**Comment by BadlyDrawnRhino** [comment]
So just to give an update, I went in and tried to backup and restore a Let's Go save this morning, and... it worked exactly as it's supposed to. So I'm not sure what happened last time. Let me know if you do want me to upload my JKSV folder, or whether you want to just leave it unless someone else reports the issue. Could have just been a random bug I suppose?

**Comment by fennectech** [comment]
We could chock it up to a single event upset.   Rarely a cosmic ray will interact with an atom in the silicon creating a shower of electronics that is /just/ enough to flip a bit.   If this happened when enumerating files a bit flip could trigger this exact thing.    We call this a single event upset.    Its entirely unpredictable and temporary event that can only be solved with ECC memory.    Which the switch lacks.     In layman's terms. "Sometimes a glitch happens that is not the fault of software or hardware cuz radiation from space hits the memory chip just right"

**Comment by J-D-K** [comment]
I'm going to close this. I have a suspicion it might have been a config bug I fixed recently.
